src/AlertBox/AlertBox.jsx

```diff
@@ -41,13 +43,15 @@ export default class AlertBox extends PureComponent {
     }
     return (
       <div className={classnames(`AlertBox--${type}`, CLASSNAMES.CONTAINER, className)}>
-        <div className={CLASSNAMES.HEADER}>
-          <span><Icon /></span>
-          <strong className={CLASSNAMES.TITLE}> {title} </strong>
-          {isClosable &&
-            <button className={CLASSNAMES.CLOSE} onClick={() => this.closeBox()}><CloseIcon /></button>
-          }
-        </div>
+        <FlexBox className={CLASSNAMES.HEADER} alignItems={ItemAlign.CENTER}>
+          <Icon />
+          <FlexBox className={CLASSNAMES.TITLE} grow>{title}</FlexBox>
```

Comment:
Should this be FlexItem? Any reason to make it a FlexBox
http://clever.github.io/components/#/components/flex-box

Reply:
^ Discussed offline, changing to a FlexItem